include/eve/module/core/regular/impl/load.hpp

```diff
@@ -110,7 +110,7 @@ namespace eve::detail
 
 #if defined(SPY_COMPILER_IS_CLANG) or defined(SPY_COMPILER_IS_GCC)
 #define DISABLE_SANITIZERS __attribute__((no_sanitize_address)) __attribute__((no_sanitize_thread))
-#elif defined(SPY_COMPILER_IS_MSVC)
+#elif defined(SPY_COMPILER_IS_MSVC) and !defined(__clang__)
```

Comment:
How is `defined(__clang__)` but not SPY_COMPILER_IS_CLANG ?
If it's clang cl, it should probably use the first DISABLE_SANITIZERS then?

Reply:
changed the ordering to be simpler